.gitignore

```diff
@@ -11,9 +11,6 @@
 # production
 /build
 
-# configurations
-/ci/ci-server.config.js
```

Comment:
This change will break the CI Server.
`/ci/ci-server.js` should be present in the gitignore